database.js

```diff
@@ -96,6 +96,10 @@ module.exports = {
         const databaseURI = postgresCS || process.env.databaseuri;
 
         if (databaseURI) {
+            //logging is enabled by default
+            if(!logging) {
```

Comment:
Asides from the fact you could probably just do a `dbConfig.logging = !!logging`, I'd agree with @sacdallago.

Logging should **never** be controlled directly by code. The best way would be a verbosity setting by command line or environment variable.